web/src/app/documentation/IntegrationKeys.md

```diff
@@ -9,7 +9,7 @@
 | `token`   | **Required** | The integration key to use.                                                                                                                                         |
 | `summary` | **Required** | Short description of the alert sent as SMS and voice.                                                                                                               |
 | `details` | _optional_   | Additional information about the alert, supports markdown.                                                                                                          |
-| `action`  | _optional_   | If set to `close`, it will close any matching alerts.                                                                                                               |
+| `action`  | _optional_   | If set to `close`, it will close any matching alerts (`summary` AND `details` must be identical).                                                                   |
 | `dedup`   | _optional_   | All calls for the same service with the same `dedup` string will update the same alert (if open) or create a new one. Defaults to using summary & details together. |
```

Comment:
Maybe clarify `dedup` and have action indicate to see below/`dedup` for details on how matching works. 

Summary and details are ignored if a user-specified `dedup` parameter is provided. If empty, then one is generated from both `summary` and `details` together.